benchalerts/tests/unit_tests/mocked_responses/GET_conbench_compare_runs_some_baseline_some_contender.json

```diff
@@ -2,81 +2,109 @@
-            "baseline_run_id": "some-run-uuid-1",
-            "baseline_z_improvement": false,
-            "baseline_z_regression": false,
-            "baseline_z_score": null,
-            "batch": "file-read",
-            "benchmark": "snappy, nyctaxi_sample, parquet, arrow",
-            "change": "0.000%",
-            "contender": "0.03637 s",
-            "contender_batch_id": "some-batch-uuid-2",
-            "contender_error": null,
-            "contender_id": "some-benchmark-uuid-3",
-            "contender_run_id": "some-run-uuid-2",
-            "contender_z_improvement": false,
-            "contender_z_regression": true,
-            "contender_z_score": null,
-            "improvement": false,
-            "language": "Python",
+            "unit": "s",
             "less_is_better": true,
-            "regression": true,
-            "tags": {
-                "compression": "snappy",
-                "cpu_count": 2,
-                "dataset": "nyctaxi_sample",
-                "file_type": "parquet",
-                "input_type": "arrow",
-                "name": "file-read"
+            "baseline": {
+                "benchmark_name": "file-read",
+                "case_permutation": "snappy, nyctaxi_sample, parquet, arrow",
+                "language": "Python",
+                "value": "0.03637 s",
+                "error": null,
+                "benchmark_result_id": "some-benchmark-uuid-1",
+                "batch_id": "some-batch-uuid-1",
+                "run_id": "some-run-uuid-1",
+                "tags": {
+                    "compression": "snappy",
+                    "cpu_count": 2,
+                    "dataset": "nyctaxi_sample",
+                    "file_type": "parquet",
+                    "input_type": "arrow",
+                    "name": "file-read"
+                }
             },
-            "threshold": "5.000%",
-            "threshold_z": 5,
-            "unit": "s"
+            "contender": {
+                "benchmark_name": "file-read",
+                "case_permutation": "snappy, nyctaxi_sample, parquet, arrow",
+                "language": "Python",
+                "value": "0.03637 s",
+                "error": null,
+                "benchmark_result_id": "some-benchmark-uuid-3",
+                "batch_id": "some-batch-uuid-2",
+                "run_id": "some-run-uuid-2",
+                "tags": {
+                    "compression": "snappy",
+                    "cpu_count": 2,
+                    "dataset": "nyctaxi_sample",
+                    "file_type": "parquet",
+                    "input_type": "arrow",
+                    "name": "file-read"
+                }
+            },
+            "analysis": {
+                "pairwise_percent_change": "0.000%",
+                "pairwise_percent_threshold": "5.000%",
+                "pairwise_regression_indicated": true,
+                "pairwise_improvement_indicated": false,
```

Comment:
Ha! Where can I read more about `pairwise`?

Reply:
`pairwise` isn't new, it's just naming what the compare endpoint already does comparing baseline and contender directly to calculate percent change. Austin suggested the naming live, and I liked it given it indicates these numbers are in the context of these two results.

I like the idea of nesting analyses; I'll keep going to do that.

Reply:
> Austin suggested the naming live, and I liked it given it indicates these numbers are in the context of these two results.

I like that, too! Cool, thanks for clarifying.